PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -213,6 +213,35 @@ def nanoAOD_recalibrateMETs(process,isData):
     process.metTables += process.corrT1METJetTable
 #    makePuppiesFromMiniAOD(process,True) # call this before in the global customizer otherwise it would reset photon IDs in VID
     runMetCorAndUncFromMiniAOD(process,isData=isData,metType="Puppi",postfix="Puppi",jetFlavor="AK4PFPuppi")
```

Comment:
Shouldn't this line be commented? This could likely explain @mariadalfonso 's observation (no difference in raw puppi met before/after the fix)

Reply:
right, it's removed in master as now the runMetCorAndUncFromMiniAOD is called 5 lines after

Reply:
ups sorry, my bad. I'll fix it now